converter.go

```diff
@@ -84,16 +75,27 @@ func resizeGeneric(in image.Image, out *image.NRGBA64, scale float64, coeffs []i
 			}
 
 			offset := (y-newBounds.Min.Y)*out.Stride + (x-newBounds.Min.X)*8
-			value := clampUint16(rgba[0] / sum)
+			// Reverse alpha-premultiplication
+			r := rgba[0] / sum
+			g := rgba[1] / sum
+			b := rgba[2] / sum
+			a := rgba[3] / sum
+
+			if a != 0 {
+				r = r * 0xffff / a
+				g = g * 0xffff / a
+				b = b * 0xffff / a
+			}
+			value := clampUint16(r)
```

Comment:
Using an output image type with pre-multiplied alpha, would save us from reversing the alpha-premultiplication here.